travis/test_pylint

```diff
@@ -22,11 +23,13 @@ expected_errors = int(
     os.environ.get('PYLINT_EXPECTED_ERRORS', 0))
 
 exit_status = 0
-if count_errors != expected_errors:
-    print("pylint expected errors {expected_errors}, "
+if count_errors == -1:
```

Comment:
Then, I'm adding a validation to get the case of `empty-folder` (-1)
I can't to use a global negative value to `empty-folder`
Maybe in future you add a new negative error:
`-2: Other handled error that no is empty folder`